docker/README.md

```diff
@@ -27,19 +48,49 @@ mkdir -p data/server
 ```
 
 4. Start the service
- 
+
 ```bash 
 USER_ID=$(id -u) GROUP_ID=$(id -g) docker-compose up -d
 ```
 
-# AWS EC2 Setup
+Congratulations! 
+Check to see if everything is working by visiting `HOSTNAME/ping`
```

Comment:
Could you give a quick example?

Reply:
Let's be more specific: it should be `https://api.${HOST_DOMAIN}/ping` for the user-facing API. This will only work, however, if an A record exists for the `api` subdomain, which we don't have explicitly in our instructions.